nautobot/extras/models/metadata.py

```diff
@@ -202,6 +202,7 @@ class ObjectMetadata(ChangeLoggedModel, BaseModel):
 
     class Meta:
         ordering = ["metadata_type"]
+        verbose_name_plural = "Object Metadata"
```

Comment:
Nit: normally verbose_name_plural is lowercase unless there's acronyms or proper nouns present (`device families`, `IP addresses`, `Git repositories`, etc.)